includes/Core/Authentication/Authentication.php

```diff
@@ -304,12 +304,17 @@ function () {
 
 		add_action(
 			'googlesitekit_authorize_user',
-			function () {
+			function ( $args ) {
```

Comment:
See previous comments about how this should change, but this function should receive 3 separate parameters rather than a single array.

Reply:
👍